flow-server/src/main/resources/webpack.generated.js

```diff
@@ -174,7 +174,7 @@ const createServiceWorkerPlugin = function () {
 const flowFrontendThemesFolder = path.resolve(flowFrontendFolder, 'themes');
 const themeOptions = {
   devMode: devMode,
-  // The following matches folder 'target/flow-frontend/themes/'
+  // The following matches folder 'frontend/generated/jarResources/themes/'
```

Comment:
```suggestion
  // The following matches folder 'frontend/generated/jar-resources/themes/'
```
So we don't confuse with the comment not matching the actual